src/common/utils.ts

```diff
@@ -105,20 +106,32 @@ function xrpToDrops(xrp: BigNumber.Value): string {
     .toString(10)
 }
 
-function toRippledAmount(amount: Amount): RippledAmount {
+function toRippledAmount(amount: Amount | RippledAmount): RippledAmount {
+  if (typeof amount === 'string')
```

Comment:
Yeah, 
```ts
type RippledAmount = Amount | string
```

So it'd actually work the same to do:
```ts
toRippledAmount(amount: RippledAmount): RippledAmount
```

I'm happy to change it, if you think that's appropriate.

Reply:
Yeah, just doing RippledAmount makes sense to me then. (My thought process was that if there was a check somewhere for explicitly "Is this an account?" it might make sense to still highlight that as an input type, but since the type never comes into question after this having just one type seems cleaner)

Reply:
bit of a nit - i don't love the omission of `{` for single-line `if` statements. will try and enforce in linter later